plugin/build_baron.go

```diff
@@ -149,28 +46,13 @@ func (bbp *BuildBaronPlugin) GetPanelConfig() (*PanelConfig, error) {
 	}, nil
 }
 
-// IsWebhookConfigured webhook will can be retrieved from project or admin config depending on PluginAdminPageDisabled flag
-// if deriving from project config, we first try to retrieve webhook config prom project parser config, otherwise we fallback to project page settings
-// version is needed to retrieve last good project config, if version is not available/empty when calling this function we must first retrieve it
+// IsWebhookConfigured retrieves webhook configuration from the project settings.
 func IsWebhookConfigured(project string, version string) (evergreen.WebHook, bool, error) {
```

Comment:
I think all of these functions should be refactored to be in the model package, since build_baron isn't really a plugin anymore but a project setting.

Reply:
Moved to project_ref.go